README.md

```diff
@@ -0,0 +1,176 @@
+# nchelpers
+This module contains the `CFDataset` class, which extends the `netcdf4.Dataset`
+class to provide additional properties, memory-efficient data access, and improved
+error handling for netCDF files that comply with the [CF Metadata Conventions](http://cfconventions.org/)
+and the PCIC metadata conventions that extend them.
+
+It supports several PCIC tools that work with netCDF files that adhere to the 
+CF and PCIC metadata conventions. The class provides several properties that
+specify information abotu a file's contents and metadata and can be used to
+guide data processing.It does not provide any new tools to directly
+modify netCDF files, but all file-modifying procedures in the netcdf4.Dataset
+class are still available.
+
+## Data chunking
+`iteration.py` contains generators for iterating over a netCDF file and loading
+on chunk at a time so that enormous files can be read without a `MemoryError`.
+
+## PCIC Metadata Model
+PCIC has a process-oriented metadata model.
```

Comment:
Add hyperlink to Confluence page? Not useful to anyone outside PCIC, but then again this whole thing is for us.